src/applications/bmqstoragetool/m_bmqstoragetool_searchresult.h

```diff
@@ -903,9 +913,39 @@ class SummaryProcessor : public SearchResult {
     GuidsSet d_partiallyConfirmedGuids;
     // Set of message guids. Messages stored here have at leas one confirmation
     // message and no delete message associated with them.
+
+    bsls::Types::Uint64 d_totalRecordsCount;
+    // Counter of total number of records.
```

Comment:
`counter` and `number` mean the same thing here, simply:
```suggestion
    // The total number of records.
```